src/transformState.js

```diff
@@ -6,6 +6,31 @@
  */
 function transformState(state, actions) {
   // write code here
+  const _addProperties = 'addProperties';
+  const _removeProperties = 'removeProperties';
+  const _clear = 'clear';
+
+  for (const action of actions) {
+    const { type, extraData, keysToRemove } = action;
+
+    switch (type) {
+      case _addProperties:
+        Object.assign(state, extraData);
+        break;
+      case _removeProperties:
```

Comment:
Add spaces for better code readability 
```suggestion
        break;
        
      case _removeProperties:
```